### DISCUSSION TITLE
Need help better synced lyrics doesn't work

### DISCUSSION THREAD

**Discussion by vaniculu** [pear-devs/pear-desktop#2773] [Q&A] [answered]
<img width="661" alt="Screenshot 2024-12-25 213735" src="https://github.com/user-attachments/assets/adcf8514-7631-421a-8ee8-dedf8c5c9963" />
<img width="919" alt="Screenshot 2024-12-25 213832" src="https://github.com/user-attachments/assets/d2a31f7d-c9e8-40fa-9686-e8f24a1333c2" />
Please help

**Comment by Lockszmith-GH** [comment, marked as answer]
The plugin is called Synced Lyrics - it's at the bottom.
Disable the Lyrics Genius one - it's the legacy one.

My guess... when Synced Lyrics will prove successful, this old one (from your screenshot) will be removed.

**Reply by vaniculu** [reply]
Yes, it works after i followed your instruction. Thank you so much :)
